<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="dataset_view">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>HSE Sans Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ДАТАСЕТ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graphic_report">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>HSE Sans Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ГРАФИЧЕСКИЕ ОТЧЕТЫ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="text_reports">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>HSE Sans Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ТЕКСТОВЫЕ ОТЧЕТЫ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
